open-machine-learning-jupyter-book/prerequisites/python-programming-introduction.md

```diff
@@ -207,74 +207,119 @@ In Python, there are several types of data types. Let us get started with the mo
 - Complex.
     Example: 1 + j, 2 + 4j
 
+
+
+**Example:**
+```{code-cell}
```

Comment:
Please format all the python code.

Reply:
I have format all the code in vscode, if you mean print('123') need change to print '123', the latter does not compile successfully in python tutor

Reply:
No. Not modify the code but just use black to format it. This may be difficult inside a MD file. Let's do this later after we convert the md to ipynb. You can skip this